<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="mainWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>471</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="inputNick">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>닉네임을 입력해주세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>460</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>입장하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adminBtn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>650</y>
       <width>88</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>관리자</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>461</width>
       <height>661</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>접속자</string>
      </attribute>
      <widget class="QTableWidget" name="loginUserList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>441</width>
         <height>571</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>접속자명</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="reloadUserList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>새로고침</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>대화목록</string>
      </attribute>
      <widget class="QPushButton" name="createChatBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>채팅방 생성</string>
       </property>
      </widget>
      <widget class="QPushButton" name="connectChatBtn">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>초대코드로 입장</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reloadChatList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>새로고침</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="chatList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>441</width>
         <height>531</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>채팅방 이름</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>대화</string>
      </attribute>
      <widget class="QPushButton" name="quitBtn">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>나가기</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>441</width>
         <height>531</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="chatMsg">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>590</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendBtn">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>590</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>88</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>방정보</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="userInfotxt_nick">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>닉네임최대로</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userInfotxt_ip">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>255.255.255.255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>(                                       )</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>tabWidget</zorder>
    <zorder>userInfotxt_nick</zorder>
    <zorder>userInfotxt_ip</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="goMainBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
